perllib/FixMyStreet/Cobrand/Bromley.pm

```diff
@@ -477,6 +477,28 @@ sub bin_services_for_address {
     my $result = $echo->GetServiceUnitsForObject($property->{uprn});
     return [] unless $result;
 
+    my $events = $echo->GetEventsForObject($property->{id});
+    my $open;
+    foreach (@$events) {
+        next if $_->{ResolvedDate} || $_->{ResolutionCodeId}; # XXX?
+        my $event_type = $_->{EventTypeId};
+        my $service_id = $_->{ServiceId};
+        if ($event_type == 2104) { # Request
+            my $data = $_->{Data}{ExtensibleDatum};
+            my $container;
+            foreach (@$data) {
+                if ($_->{ChildData}) {
+                    foreach (@{$_->{ChildData}{ExtensibleDatum}}) {
+                        $container = $_->{Value} if $_->{DatatypeName} eq 'Container Type';
```

Comment:
does it make sense to have a `last` here once you match container? I assume there can be only one.